### ISSUE TITLE
Enable HIDAPI feature of SDL2

### ISSUE THREAD

**Issue by hikari-no-yume** [hikari-no-yume/touchHLE#85]
This is trivial to do: add `"hidapi"` to the list of `rust-sdl2` features in `Cargo.toml`.

We'll need this for accelerometer support: https://github.com/hikari-no-yume/touchHLE/issues/13. It might also improve controller support for some OSes?

I don't want to enable this just yet, however, because there's an unfortunate issue on macOS with controllers getting duplicated: https://github.com/libsdl-org/SDL/issues/7479. Either that needs to be fixed upstream, or we need to work around it somehow. I'd like to avoid pushing that problem onto the user if possible (touchHLE is already tricky to use).

**Comment by hikari-no-yume** [comment]
15a960102f3db6fe60948e72ff6e567128f32bad enables the `"hidapi"` feature, but a hint is set so that the actual HIDAPI drivers still aren't used, so this issue is still applicable.

**Comment by mubinulhaque** [comment]
I believe SDL has fixed the issue now so there should be no issue dealing with HIDAPI now, right?

**Comment by hikari-no-yume** [comment]
Yes, that's right, I just need to get around to it… I will need to fork Rust-SDL2 again to update the commit hash.
